### ISSUE TITLE
Use a consistent size for testharness.js

### ISSUE THREAD

**Issue by gsnedders** [w3c/web-platform-tests#7642]
See https://bugs.webkit.org/show_bug.cgi?id=178053

We guarantee a given size for reftests (well, modulo #7598), and not doing so appears to be causing problems for some testharness.js tests, so we should probably just guarantee a size for them too.

cc/ @youennf @cdumez [I can't find any GitHub account for Darin?] @jgraham @Ms2ger @foolip

**Comment by foolip** [comment]
Makes sense. What should the size be?

**Comment by gsnedders** [comment]
I suggest we go for the same as we decide in #7598 to be consistent everywhere.

**Comment by foolip** [comment]
https://chromium-review.googlesource.com/c/chromium/src/+/768430 is a review where this is a problem. What is the size we'd guarantee, is it 800x600?

**Comment by Ms2ger** [comment]
I think 600×600, but I thought that was a minimum, not a guaranteed size. I'm definitely not using that when running tests in the browser manually. Can that dialog test compute the expected result dynamically?

**Comment by foolip** [comment]
Yes, the dialog test could be written to work with any viewport size above some minimum size. But that would amount to saying that this issue is WontFix, and tests having to deal with it.

**Comment by gsnedders** [comment]
Practically this is nowadays RFC material, and see https://github.com/web-platform-tests/rfcs/issues/188 for an issue on the RFCs repo.
